srcs/window.c

```diff
@@ -6,16 +6,19 @@
 /*   By: melogr@phy <tgrivel@student.42lausanne.ch  +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/08/17 22:01:59 by melogr@phy        #+#    #+#             */
-/*   Updated: 2022/09/05 23:40:09 by melogr@phy       ###   ########.fr       */
+/*   Updated: 2022/09/10 18:39:30 by gudias           ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include	"cub3d.h"
 
 // If the ESC key is pressed, the window closes
 static int
-	deal_key(int key, void *info)
+	deal_key(int key, void *params)
 {
+	t_info	*info;
+
+	info = params;
```

Comment:
I already tried it, that work
> commit message if we commit this change: `window.c: cast the parameters to t_info`
```suggestion
	deal_key(int key, t_info *info)
{
```